exporter/opentelemetry-exporter-prometheus-remote-write/src/opentelemetry/exporter/prometheus_remote_write/__init__.py

```diff
@@ -159,10 +162,16 @@ def headers(self, headers: Dict):
     def export(
         self, export_records: Sequence[ExportRecord]
     ) -> MetricsExportResult:
-        raise NotImplementedError()
+        timeseries = self._convert_to_timeseries(export_records)
+        if not timeseries:
+            logger.warning("No valid records found, export aborted")
+            return MetricsExportResult.FAILURE
```

Comment:
Is it really a failure of there are no records to be exported?

Reply:
This would only occur if all provided records contain only unsupported aggregators, so I figured it would better to signal it as a failure than continuously exporting nothing despite receiving records. I am adding a check at the start to return success if no records I provided to handle that valid case.

Reply:
Also changed the log to an error and made issue more clear in message.